extensions/amp-brightcove/0.1/amp-brightcove.js

```diff
@@ -100,13 +100,6 @@ class AmpBrightcove extends AMP.BaseElement {
     return isLayoutSizeDefined(layout);
   }
 
-  /** @override */
-  viewportCallback(visible) {
-    this.element.dispatchCustomEvent(VideoEvents.VISIBILITY, {
```

Comment:
This is blocked on the video PR, right? We need to make sure we don't merge in the wrong order.